pywbem/cim_operations.py

```diff
@@ -1709,6 +1721,269 @@ def OpenEnumerateInstancePaths(self, ClassName, namespace=None,
+
+            * If not `None`, this parameter is sent to the WBEM server as the
+              proposed timeout for the enumeration session. A value of 0
+              indicates that the server is expected to never time out. The
+              server may reject the proposed value, causing a
+              :class:`~pywbem.CIMError` to be raised with status code
+              :attr:`~pywbem.CIM_ERR_INVALID_OPERATION_TIMEOUT`.
+            * If `None`, this parameter is not passed to the WBEM server, and
+              causes the server-implemented default timeout to be used.
+
+          ContinueOnError (:class:`py:bool`):
+            Indicates to the WBEM server to continue sending responses
+            after an error response has been sent.
+
+            * If `True`, the server is to continue sending responses after
+              sending an error response. Not all servers support continuation
+              on error; a server that does not support it must send an error
+              response if `True` was specified, causing
+              :class:`~pywbem.CIMError` to be raised with status code
+              :attr:`~pywbem.CIM_ERR_CONTINUATION_ON_ERROR_NOT_SUPPORTED`.
+            * If `False`, the server is requested to close the enumeration after
+              sending an error response.
+            * If `None`, this parameter is not passed to the WBEM server, and
+              causes the server-implemented default behaviour to be used.
+              :term:`DSP0200` defines that the server-implemented default is
+              `False`.
+
+          MaxObjectCount (:class:`~pywbem.Uint32`)
+            Maximum number of instances the WBEM server may return
+            for each of the open and pull requests issues as part of the
+            execution of this generator.
+
+            * If positive, the WBEM server is to return no more than the
+              specified number of instances.
+            * Zero is not allowed in that it would mean that zero instances
+              are to be returned for every request issued
+            * If `None`, this parameter is not passed to the WBEM server, and
+              causes the server-implemented default behaviour to be used.
+              :term:`DSP0200` defines that the server-implemented default is
+              to return zero instances.
+
+        Keyword Arguments:
+
+          extra :
+            Additional keyword arguments are passed as additional operation
+            parameters to the WBEM server.
+            Note that :term:`DSP0200` does not define any additional parameters
+            for this operation.
+
+        Raises:
+
+            Exceptions described in :class:`~pywbem.WBEMConnection`.
+
+        Returns:
+
+          A Python iterator object. Instances can be retrieved
+          by iterating through the iterator.
+
+        Example:
+            enum_gen = conn.EnumerateInstancesWithPathGen('CIM_Blah')
```

Comment:
- The function is now called differently.
- The return value is an iterator not a generator and the variable storing it should not end with `_gen`. Maybe just  `instances`?